pokedex/data/csv/items.csv

```diff
@@ -1085,3 +1085,524 @@ id,identifier,category_id,cost,fling_power,fling_effect_id
+1497,★sco5953,,20,,
+1498,★sco5984,,20,,
+1499,★sco6508,,20,,
+1500,★sco6084,,20,,
+1501,★sco5944,,20,,
+1502,★sco6630,,20,,
+1503,★sco6027,,20,,
+1504,★sco6247,,20,,
+1505,★sco6252,,20,,
+1506,★sco5928,,20,,
+1507,★sco6241,,20,,
+1508,★sco6165,,20,,
+1509,★tri544,,20,,
+1510,★leo3982,,20,,
+1511,★leo4534,,20,,
+1512,★leo4357,,20,,
+1513,★leo4057,,20,,
+1514,★leo4359,,20,,
+1515,★leo4031,,20,,
+1516,★leo3852,,20,,
+1517,★leo3905,,20,,
+1518,★leo3773,,20,,
+1519,★gru8425,,20,,
+1520,★gru8636,,20,,
+1521,★gru8353,,20,,
+1522,★lib5685,,20,,
+1523,★lib5531,,20,,
+1524,★lib5787,,20,,
+1525,★lib5603,,20,,
+1526,★pup3165,,20,,
+1527,★pup3185,,20,,
+1528,★pup3045,,20,,
+1529,★cyg7924,,20,,
+1530,★cyg7417,,20,,
+1531,★cyg7796,,20,,
+1532,★cyg8301,,20,,
+1533,★cyg7949,,20,,
+1534,★cyg7528,,20,,
+1535,★oct7228,,20,,
+1536,★col1956,,20,,
+1537,★col2040,,20,,
+1538,★col2177,,20,,
+1539,★gem2990,,20,,
+1540,★gem2891,,20,,
+1541,★gem2421,,20,,
+1542,★gem2473,,20,,
+1543,★gem2216,,20,,
+1544,★gem2777,,20,,
+1545,★gem2650,,20,,
+1546,★gem2286,,20,,
+1547,★gem2484,,20,,
+1548,★gem2930,,20,,
+1549,★peg8775,,20,,
+1550,★peg8781,,20,,
+1551,★peg39,,20,,
+1552,★peg8308,,20,,
+1553,★peg8650,,20,,
+1554,★peg8634,,20,,
+1555,★peg8684,,20,,
+1556,★peg8450,,20,,
+1557,★peg8880,,20,,
+1558,★peg8905,,20,,
+1559,★oph6556,,20,,
+1560,★oph6378,,20,,
+1561,★oph6603,,20,,
+1562,★oph6149,,20,,
+1563,★oph6056,,20,,
+1564,★oph6075,,20,,
+1565,★ser5854,,20,,
+1566,★ser7141,,20,,
+1567,★ser5879,,20,,
+1568,★her6406,,20,,
+1569,★her6148,,20,,
+1570,★her6410,,20,,
+1571,★her6526,,20,,
+1572,★her6117,,20,,
+1573,★her6008,,20,,
+1574,★per936,,20,,
+1575,★per1017,,20,,
+1576,★per1131,,20,,
+1577,★per1228,,20,,
+1578,★per834,,20,,
+1579,★per941,,20,,
+1580,★phe99,,20,,
+1581,★phe338,,20,,
+1582,★vel3634,,20,,
+1583,★vel3485,,20,,
+1584,★vel3734,,20,,
+1585,★aqr8232,,20,,
+1586,★aqr8414,,20,,
+1587,★aqr8709,,20,,
+1588,★aqr8518,,20,,
+1589,★aqr7950,,20,,
+1590,★aqr8499,,20,,
+1591,★aqr8610,,20,,
+1592,★aqr8264,,20,,
+1593,★cru4853,,20,,
+1594,★cru4730,,20,,
+1595,★cru4763,,20,,
+1596,★cru4700,,20,,
+1597,★cru4656,,20,,
+1598,★psa8728,,20,,
+1599,★tra6217,,20,,
+1600,★cap7776,,20,,
+1601,★cap7754,,20,,
+1602,★cap8278,,20,,
+1603,★cap8322,,20,,
+1604,★cap7773,,20,,
+1605,★sge7479,,20,,
+1606,★car2326,,20,,
+1607,★car3685,,20,,
+1608,★car3307,,20,,
+1609,★car3699,,20,,
+1610,★dra5744,,20,,
+1611,★dra5291,,20,,
+1612,★dra6705,,20,,
+1613,★dra6536,,20,,
+1614,★dra7310,,20,,
+1615,★dra6688,,20,,
+1616,★dra4434,,20,,
+1617,★dra6370,,20,,
+1618,★dra7462,,20,,
+1619,★dra6396,,20,,
+1620,★dra6132,,20,,
+1621,★dra6636,,20,,
+1622,★cvn4915,,20,,
+1623,★cvn4785,,20,,
+1624,★cvn4846,,20,,
+1625,★aql7595,,20,,
+1626,★aql7557,,20,,
+1627,★aql7525,,20,,
+1628,★aql7602,,20,,
+1629,★aql7235,,20,,
+1630,max-honey,,4000,30,
+1631,max-mushrooms,,6000,30,
+1632,galarica-twig,,20,30,
+1633,galarica-cuff,,3000,30,
+1634,style-card,,0,,
+1635,armor-pass,,0,,
+1636,rotom-bike,,0,,
+1637,rotom-bike,,0,,
```

Comment:
more rotom bikes. merge them or give them a unique identifier (ala bike--green and bike--yellow).

Reply:
Those two are `rotom-bike--sparkling-white` and `rotom-bike--glistering-black`